.github/workflows/update_catalog.yml

```diff
@@ -0,0 +1,54 @@
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "Catalog/catalog.json"
```

Comment:
@chusloj One other thing: you may want to add `.github/workflows/update_catalog.yml` to the list of files that triggers a build

Reply:
Will do!

Reply:
Just implemented this.